cmd/discovery5.go

```diff
@@ -103,5 +128,15 @@ func RunDiscv5(ctx *cli.Context) error {
 	// Discovery5 service
 	// check: https://github.com/migalabs/armiarma/blob/ca3d2f6adea364fc7f38bdabda912b5541bb4154/src/discovery/dv5/dv5_service.go#L58
 	// Bootnodes are in pkg/config/bootnodes
+	c := cli.Context
+	infObj, _ := info.InitEth2(c)
+	dv5, _ := dv5.NewDiscovery(
+		ctx.Context,
+		ln, // LocalNode?
+		(*ecdsa.PrivateKey)(infObj.PrivateKey),
```

Comment:
Seems that we don't need such a complex casting: 
you could just use the `*ecdsa.Privatekey` from the function `GenNewPrivKey()`